packages/lesswrong/server/callbacks/userCallbacks.tsx

```diff
@@ -77,6 +77,7 @@ getCollectionHooks("Users").editSync.add(function maybeSendVerificationEmail (mo
       && (!user.whenConfirmationEmailSent
           || user.whenConfirmationEmailSent.getTime() !== modifier.$set.whenConfirmationEmailSent.getTime()))
   {
+    // Note: This is still gated by verifyEmailsSetting because `canUpdate` of whenConfirmationEmailSent is controlled by verifyEmailsSetting
     void sendVerificationEmail(user);
```

Comment:
Why not just be safe and change this to `sendVerificationEmailConditional()`?

Reply:
Seems reasonable 👍